test/operators.jl

```diff
@@ -89,3 +89,9 @@ end
     @test filter(!isupper, str) == replace(str, r"[A-Z]", "")
     @test filter(!islower, str) == replace(str, r"[a-z]", "")
 end
+
+# issue #19891
+B = 0 .< [1 -1 5] .< 3
+@test B == [true false false]
+B = 3 .> [1 -1 5] .> 0
+@test B == [true false false]
```

Comment:
I guess Travis doesn't check for this, but it's usually a good idea to have a newline character at the end of a text file.